origin-dapp/src/components/conversation-list-item.js

```diff
@@ -1,20 +1,119 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
-
+import moment from 'moment'
 import Avatar from 'components/avatar'
+import { getListing } from 'utils/listing'
+import { defineMessages, injectIntl } from 'react-intl'
 
 class ConversationListItem extends Component {
+  constructor(props) {
+    super(props)
+    const { selectedLanguageCode } = this.props
+
+    this.intlMessages = defineMessages({
+      s: {
+        id: 'messages.moment.sec',
+        defaultMessage: 'Now'
+      },
+      ss: {
+        id: 'messages.moment.seconds',
+        defaultMessage: 's'
+      },
+      min: {
+        id: 'messages.moment.minutes',
+        defaultMessage: 'min'
+      },
+      hour: {
+        id: 'messages.moment.hours',
+        defaultMessage: 'h'
+      },
+      day: {
+        id: 'messages.moment.days',
+        defaultMessage: 'd'
+      },
+      week: {
+        id: 'messages.moment.week',
+        defaultMessage: 'wk'
+      },
+      month: {
+        id: 'messages.moment.Months',
```

Comment:
Is there a reason to capitalize this? 📆

Reply:
I think this is the remnants of trying to differentiate minutes from months. I'll update this